stdlib/contextlib.pyi

```diff
@@ -37,6 +37,10 @@ if sys.version_info < (3,):
 class closing(ContextManager[_T]):
     def __init__(self, thing: _T) -> None: ...
 
+if sys.version_info >= (3, 10):
+    class aclosing(AsyncContextManager[_T]):
```

Comment:
Can we bound T to a Protocol that defines `async def aclose`?

The same could be done for `closing`.